app/docs/enterprise/0.31-x/installation/ubuntu.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to install Kong Enterprise and PostgreSQL onto Ubuntu
+---
+
+## Install Kong Enterprise
+
+Download the EE binary for Ubuntu (or Debian) from Bintray
+SFTP the file to your Ubuntu server
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install openssl libpcre3 procps perl
+$ sudo dpkg -i kong-enterprise-edition-0.31-1.zesty.all.deb
+```
+NOTE: EE file may differ in last step above
```

Comment:
We support other distributions of Ubuntu too, I think it would be worth mentioning them?